dev-install-locally-no-tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+./gradlew clean install -x test
```

Comment:
We do have existing shell scripts. Could you just add install to the list of commands and remove this file?